.github/workflows/ci.yml

```diff
@@ -228,9 +234,16 @@ jobs:
       - dart-analyze
       - dart-test
       - dart-doc
+    if: ${{always()}}
     runs-on: ubuntu-20.04
     steps:
-      - run: exit 0
+      - run: |
```

Comment:
Here it has to be done this way because `dart-ok` depends on  `dart-test` and this one depends on `cargo-test`. Something like:

` cargo-test -> dart-test -> dart-ok ` 

 Then if `cargo-test` doesn't run, `dart-test` has a condition of `if: ${{always()` to run always and then check whether `cargo-test` ran or not. But because `cargo-test` got skipped and `dart-ok` has some type of dependency with `cargo-test` then it will skip as well, so it has to have the `if: ${{always()` as well and the `needs:` won't work anymore. So the best way I found was checking if the jobs were successful.